### PR TITLE
Add empty arrays for Headers, Params instead of missing json objects

### LINKED ISSUE #1151: Make mappings actuator output compatible with SpringBootAdmin
Currently viewing the mappings page on spring boot admin shows a blank, with a javascript error: `Cannot read property 'length' of undefined `

## Describe the solution you'd like
Spring boot admin is looking for 'Headers', 'Params' and in general any route details to have a Json array even if empty.

### PR DESCRIPTION
**Pull request by hananiel**
## Description

Adding additional fields in Json output to support Spring Boot Admin. The fields in question 'Headers' and 'Params' are Spring boot features on @RequestMapping annotation to match routes to controller actions. In .Net this is accomplished differently and is hard to infer as easily as in Spring. So, we will just add empty arrays to cause SpringBootAdmin not to error on this page.

Fixes #1151 

## Quality checklist

- [x] Your code complies with our [Coding Style](https://github.com/SteeltoeOSS/.github/blob/main/contributing-docs/contributing-code-style.md).
- [x] You've updated unit and/or integration tests for your change, where applicable.
- [ ] You've updated documentation for your change, where applicable.
      If your change affects other repositories, such as [Documentation](https://github.com/SteeltoeOSS/Documentation), [Samples](https://github.com/SteeltoeOSS/Samples) and/or [MainSite](https://github.com/SteeltoeOSS/MainSite), add linked PRs here.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.
- [x] You've added required license files and/or file headers (explaining where the code came from with proper attribution), where code is copied from StackOverflow, a blog, or OSS.

### CODE REVIEW

File: src/Management/src/Endpoint/Mappings/MappingDescription.cs [hunk 1]
```diff
@@ -50,16 +50,22 @@ private RouteMappingDetails CreateMappingDetails(IRouteDetails routeDetails)
...
                 {
                     routeDetails.RouteTemplate
-                }
+                },
+                Params = new string[]
+                {
+                }, // does not apply for .NET
+                Headers = new string[]
+                {
+                } // Cannot infer here for .NET
```
Comment (bart-vmware) [inline src/Management/src/Endpoint/Mappings/MappingDescription.cs:68]: Why return a list in some cases, and an array in others? I suppose returning a list in all cases is slightly more convenient to calling code that wants to modify the collections.

File: src/Management/src/Endpoint/Mappings/AspNetCoreRouteDetails.cs [hunk 2]
```diff
@@ -13,4 +13,8 @@ public class AspNetCoreRouteDetails : IRouteDetails
     public IList<string> Produces { get; set; }
 
     public IList<string> Consumes { get; set; }
+
+    public IList<string> Params { get; set; }
+
+    public IList<string> Headers { get; set; }
```
Comment (bart-vmware) [inline src/Management/src/Endpoint/Mappings/AspNetCoreRouteDetails.cs:19]: Note I'm going to initialize these properties with empty collections when merging this into #1155, based on what we discussed yesterday. I already did that for all existing collection properties there, so I think it makes the most sense to handle that in my PR.

File: src/Management/src/Endpoint/Mappings/MappingDescription.cs (hunk 1 above)
Comment (hananiel) [reply src/Management/src/Endpoint/Mappings/MappingDescription.cs]: fixed to return list all the time.

**Review by bart-vmware — APPROVED** [review]
LGTM

**Pull request merged**